v2/pkg/catalog/loader/loader.go

```diff
@@ -148,6 +148,12 @@ func (store *Store) Load() {
 	store.workflows = store.LoadWorkflows(store.finalWorkflows)
 }
 
+var uniqueTemplatesIDValidateMap map[string]string
```

Comment:
not that is that super important, but I will still mention it:
* keys in a map are always unique, hence stating it in the name twice (unique + map) is redundant
* "validate" is an action, which is usually used for method names. This is a map, which only stores data and does not operate over it. I'd rather use "validation" or would just ditch it entirely
* if you only say TemplateIDMap, then I am thinking of a map used as set in golang (`map[string]interface{}{}`), so you should rather state the value part as well

Based on all the above a potentially more suitable name would be something like `templateIdPathMap`